kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeAccountDialog/ConfirmAccountDetails.vue

```diff
@@ -105,7 +105,7 @@
         },
       });
       function cleanValue(value) {
-        return value === DemographicConstants.NOT_SPECIFIED ? '' : value;
+        return value === DemographicConstants.NOT_SPECIFIED ? '' : <KEmptyPlaceholder />;
```

Comment:
This is not quite right - and seems to be assuming that we are using a JSX code style, which we are not.

The `<KEmptyPlaceholder />` should be used within the `template` section of the component where this value is being displayed, using `v-if`, for more information, see https://v2.vuejs.org/v2/guide/conditional.html